Adding a little tip about food on registration

ezefranca opened the pull request:
Last weekend I was attend a hackathon was offering vegetarian food (with cheese) saying "vegan". Because of this, I think this tip is important for organizers.

cbetta commented:
@ezefranca I like it. Can you expand on it for a bit, moving it out of the brackets and into a full sentence? I also don't think that place is the right place. Maybe we can add it as a segment under the "Sustenance" header?

ezefranca commented:
@cbetta Nice. I moved to "Sustenance" (It's be the best place to this, thanks), added a link about allergies, and a link to  [Sustenance](#sustenance) in the description.

thanks for your time :)

cbetta commented:
@ezefranca thanks for your contribution. I've taken some liberty to rewrite it though as it was messing with the layout and went a bit off topic.

I've rewritten the section to be added after the section you added to point out what veganism is in detail. We already point out that vegan meals are a great idea for a meal that can be eaten by most people, and I've added an extra paragraph pointing out what vegan food really is.

ezefranca commented:
@cbetta  Thank you 👍
